tests/e2e/env/setup.sh

```diff
@@ -310,7 +313,10 @@ echo "Creating screenshots directory"
 mkdir -p $WCP_ROOT/screenshots
 
 echo "Disabling rate limiter for card declined in E2E tests"
-cli wp option add wcpay_session_rate_limiter_disabled_wcpay_card_declined_registry yes
+cli wp option set wcpay_session_rate_limiter_disabled_wcpay_card_declined_registry yes
+
+echo "Removing all coupons ..."
+cli wp db query "DELETE p, m FROM wp_posts p LEFT JOIN wp_postmeta m ON p.ID = m.post_id WHERE p.post_type = 'shop_coupon'"
```

Comment:
Ensured that the `shop_coupon` doesn't exist before running the next line. It fails when the coupon already exists. Those changes were required when running `npm run test:e2e-setup` multiple times.

Reply:
Could we use `shop_coupon get` maybe to check and create if it doesn't exist? Maintaining raw queries could get trickier compared to standard CLI commands.